testsuite/tests/multicluster/conftest.py

```diff
@@ -176,8 +174,12 @@ def client(hostname, gateway, gateway2):  # pylint: disable=unused-argument
 
 
 @pytest.fixture(scope="module", autouse=True)
-def commit(request, routes, dns_policy, dns_policy2, tls_policy, tls_policy2):  # pylint: disable=unused-argument
-    """Commits all policies before tests"""
+def commit(
+    request, routes, dns_policy, dns_policy2, tls_policy, tls_policy2, gateway, gateway2
+):  # pylint: disable=unused-argument
+    """Waits for gateways to be ready and commits all policies before tests"""
+    gateway.wait_for_ready()
+    gateway2.wait_for_ready()
     components = [dns_policy, dns_policy2, tls_policy, tls_policy2]
```

Comment:
can you add gateways to the beginning of the `components` for commit, and with it also cleanup and commit gateway here?